Gemfile.lock

```diff
@@ -80,9 +79,6 @@ GEM
       rubocop (~> 1.41)
     rubocop-factory_bot (2.24.0)
       rubocop (~> 1.33)
-    rubocop-performance (1.19.1)
-      rubocop (>= 1.7.0, < 2.0)
-      rubocop-ast (>= 0.4.0)
```

Comment:
We might want to consider adding this gem back.

Reply:
Can do